tests/store/api/test_store_api_status_analysis.py

```diff
@@ -1,17 +1,19 @@
 """This script tests the cli methods to add families to status-db"""
 from datetime import datetime, timedelta
 
-from cg.constants import Pipeline
+from cg.constants import Pipeline, DataDelivery
 from cg.store import Store
 
 
-def test_that_many_families_can_have_one_sample_each(base_store: Store):
+def test_that_many_families_can_have_one_sample_each(base_store: Store, helpers):
```

Comment:
Should we not change this to case now and make it all the way through

Reply:
fixed